extensions/sql-migration/src/dialog/createSqlMigrationService/createSqlMigrationServiceDialog.ts

```diff
@@ -135,13 +156,45 @@ export class CreateSqlMigrationServiceDialog {
 			});
 		});
 
+		this._testConnectionButton = azdata.window.createButton(constants.TEST_CONNECTION);
+		this._testConnectionButton.hidden = true;
+		this._testConnectionButton.onClick(async (e) => {
+			this._refreshLoadingComponent.loading = true;
+			this._connectionStatus.updateCssStyles({
+				'display': 'none'
+			});
+			try {
+				await this.refreshStatus();
+			} catch (e) {
+				console.log(e);
```

Comment:
if the test connection action fails, the user won't see anything, why not show the error?